lib/fastlane/plugin/revenuecat_internal/actions/determine_next_version_using_labels_action.rb

```diff
@@ -10,8 +10,9 @@ def self.run(params)
         repo_name = params[:repo_name]
         github_token = params[:github_token]
         rate_limit_sleep = params[:github_rate_limit]
+        skip_if_no_public_changes = params[:skip_if_no_public_changes]
 
-        Helper::VersioningHelper.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep)
+        Helper::VersioningHelper.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep, skip_if_no_public_changes)
```

Comment:
Hmm I wonder, would this ever be `false`? Not sure where we would need this option.

Reply:
Good point... I think I am over optimizing here so I will remove the parameter